src/DashboardService.Models.Dto/Requests/Board/CreateBoardRequest.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace LT.DigitalOffice.DashboardService.Models.Dto.Requests.Board;
+
+public record class CreateBoardRequest
+{
+  [Required]
```

Comment:
this attribute applies to reference types, if you dont send a value for a property of value type, then the request will contain the default value for this type

Reply:
+